chaste_codegen/chaste_model.py

```diff
@@ -330,10 +339,10 @@ def _get_modifiable_parameters(self):
         if self.expose_annotated_variables:
             # Combined and sorted in display name order
             return \
-                sorted(self._get_modifiable_parameters_annotated() + self._get_modifiable_parameters_exposed(),
+                sorted(set(self._get_modifiable_parameters_annotated() + self._get_modifiable_parameters_exposed()),
```

Comment:
Which would avoid needing to cast to & from set here.

Reply:
actually no it would still be necessary because cytosolic_calcium_concentration will now show up both through the metadata and through the fact it has annotation. It should have actually always been there realy to capture this case